packages/xo-server/src/xo-mixins/metadata-backups.js

```diff
@@ -401,9 +395,6 @@ export default class metadataBackup {
 
           this._listPoolMetadataBackups(REMOVE_CACHE_ENTRY, remoteId)
         } catch (error) {
-          if (outputStream !== undefined) {
-            outputStream.destroy()
-          }
```

Comment:
We need to properly handle this.

Possible approaches:
- make `outputStream` supports a timeout
- pass a cancel token to `outputStream`
- emit an error in `stream`